config.template.json

```diff
@@ -11,5 +11,7 @@
   "clear_current_session_command": "#clear session",
   "clear_all_sessions_command": "#clear all sessions",
   "chat_group_session_independent": false,
-  "single_chat_prefix": ["bot", "@bot"]
+  "single_chat_prefix": ["bot", "@bot"],
+  "query_key_command": "#query key",
```

Comment:
Thanks for the contribution🎉. Please also add relevant description in  in README [here](https://github.com/iuiaoin/wechat-gptbot#configuration) to help others understand your configuration～

Reply:
README has been updated.

Reply:
Yes, the code is able to work properly.